docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,157 @@
+# KubeStellar Hub chart usage
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choise of user-defined Control Plains.
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.1
+```
+
+## Pre-requisites
+
+A list of executable that may be required during the deployment can be fund [here](./pre-reqs.md).
+At minimum [Helm](https://helm.sh/) is required.
+
+The setup of KubStellar via the Hub chart requires the existance of at least one cluster
+to be used for the deployment of the chart.
+
+This can be:
+
+1. A local **Kind** cluster with an ingress with SSL passthrough and a mapping to host port 9443
+
+This option is particularly useful for first time users or users that would like to have a local deployment.
+
+For convenience, a new local Kind clusters that satisfies the requirements for KubeStellar setup
+and that can be used to exercises the [examples](./examples.md) can be created with the following command:
+
+```shell
+sudo su <<EOF
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubestellar --port 9443
+EOF
+```
+
+It is important to note that, if a different name is used for the Kind cluster, then its control plane name must be also be referenced during the Helm chart installation by setting `--set "kubeflex-controller.hostContainer=<control-plane-name>".
+
+2. A local **k3s** cluster with an ingress with SSL passthrough and a mapping to host port 9443
+
+... Under onstruction ...
+
+3. An **OpenShift** cluster
+
+When using this option, one is required to explicitely set the `isOpenShift` variable to `true` by including `--set "kubeflex-operator.isOpenShift=true"` in the Helm chart installation command.
+
+
+## KubeStellar Hub Chart values
+
+The KubeStellar chart makes available to the user several values that may be used to customize its installation into an existing cluster:
+
+```yaml
+# KubeFlex override values
+kubeflex-operator:
+  install: true # enable/disable the installation of KubeFlex by the chart (default: true)
+  installPostgreSQL: true # enable/disable the installation of the appropriate version of PostgreSQL required by KubeFlex (default: true)
+  isOpenShift: false # set this variable to true when installing the chart in an OpenShift cluster (default: false)
+  # Kind cluster specific settings:
+  domain: localtest.me # used to set the Control Planes DNS in a Kind cluster installation (default: localtest.me)
+  externalPort: 9443 # used to set the port to access the Control Planes API (default: 944)

```

Comment:
Why is "(default: 944)" in the comment on this line?

Reply:
typo

Reply:
Why write the default value twice on one line?

Reply:
I guess it can be removed, in reality in the values.yaml, the above is commented out and it relies on the kubeflex chart default values